package.json

```diff
@@ -47,6 +47,8 @@
     "scripts": {
         "start": "webpack serve",
         "build": "tsc && npx ncp src/resources/ dist/resources",
+        "build:css": "postcss src/styled/tailwind.css -o src/styled/styles.css",
```

Comment:
Do we really need this?

Reply:
Yes, it is needed to transform all tailwind clases into a single static css file.